### PR TITLE
spanish support updated

### PR DESCRIPTION
**Pull request by danisaurio**
add Spanish support and updated readme information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 This repository lets you create WCAG EM Reports using [Eleventy](https://www.11ty.dev/). 
 
 * Write issues as Markdown files
-* Create reports in English or Dutch 
+* Create reports in English, Dutch, Brazilian Portuguese or Spanish
```
Comment (hidde) [inline README.md:6]: Ah, thanks for updating, forgot about this one 🙏

**Review by hidde — APPROVED** [review]
Thanks, looks good to me! 

Are you ok for it to be merged?

**Comment by danisaurio** [comment]
Sure! :D happy to contribute!

**Comment by hidde** [comment]
Thank you for your contribution, @danisaurio !

**Pull request merged**